src/components/layouts/collection-page-layout.tsx

```diff
@@ -885,6 +885,9 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
               <div>
                 <ul>
                   {lessons.map((lesson: LessonResource, index: number) => {
+                    const tagImageUrl = `https://res.cloudinary.com/dg3gyk0gu/image/upload/v1683914713/tags/${lesson?.primary_tag?.name}.png`
+                    const optimizedImageUrl = `https://res.cloudinary.com/dg3gyk0gu/image/fetch/w_72,h_72/${tagImageUrl}`
+
```

Comment:
optimized image from cloudinary

Reply:
i'm positive you can just add the transform to the first url instead of passing it to fetch